instrumentation/gruf/lib/opentelemetry/instrumentation/gruf/version.rb

```diff
@@ -7,7 +7,7 @@
 module OpenTelemetry
   module Instrumentation
     module Gruf
-      VERSION = '0.1.1'
+      VERSION = '0.1.2'
```

Comment:
```suggestion
      VERSION = '0.1.1'
```
Our release process manages gem versions. No need to change this manually

Reply:
ok